bin/build

```diff
@@ -13,6 +13,6 @@ BUILD_SCRIPT="cd /src && "\
 "rm -rf build && mkdir build && "\
 "go build -o build/codeclimate-govet ."
 
-docker run -v "$SRC_ROOT:/src" yunspace/alpine-golang /bin/sh -c "$BUILD_SCRIPT"
+docker run -v "$SRC_ROOT:/src" golang:latest /bin/sh -c "$BUILD_SCRIPT"
```

Comment:
I realize this is already shipped, but is there a reason not to use :alpine here as well? Should make builds slightly faster to only need to pull one base image.